logos/src/logos/pipeline/executor.py

```
@@ -0,0 +1,221 @@
+        }
+        
+        # Inject model name for OpenWebUI
+        if "openwebui" in context.provider_name.lower():
+            payload = {**payload, "model": context.model_name}
+        
+        payload["stream"] = True
+        payload["stream_options"] = {"include_usage": True}
+        
+        logger.info(f"Stream-Executing request to {context.forward_url} (model: {context.model_name})")
+        
+        async with httpx.AsyncClient(timeout=None) as client:
```

Ruff (0.14.8)

Probable use of `httpx` call with timeout set to `None`

(S113)